tests/bbcode/parser_test.php

```diff
@@ -249,10 +249,18 @@ public function test_bbcode_firstpass($description, $message, $expected, $incomp
 			$this->markTestIncomplete($incomplete);
 		}
 
-		global $user, $request, $symfony_request;
+		global $user, $request, $symfony_request, $phpbb_dispatcher, $config, $phpEx, $request, $symfony_request;
```

Comment:
should be:
global $user, $request, $symfony_request, $phpbb_dispatcher, $config, $phpEx;

Reply:
I've looked into the chain of methods called in deep and had to add more global vars.

Reply:
You've duplicated `$request, $symfony_request`